grpc.echo/test.sh

```diff
@@ -16,7 +16,7 @@ echo EXPECTED="$EXPECTED"
 echo
 
 # WHEN
-OUTPUT=$(docker run --network zilla-grpc-echo_default --rm -v ./echo.proto:/echo.proto fullstorydev/grpcurl -plaintext -proto echo.proto  -d "$INPUT" zilla.examples.dev:$PORT example.EchoService.EchoUnary)
+OUTPUT=$(docker run --network zilla-grpc-echo_default --rm -v ./etc/protos/echo.proto:/echo.proto fullstorydev/grpcurl -plaintext -proto echo.proto  -d "$INPUT" zilla.examples.dev:$PORT example.EchoService.EchoUnary)
```

Comment:
Can we use `docker compose run ...` here instead of `docker run`?

That should simplify and let us pick up the configuration in `compose.yaml` implicitly without needing to respecify volume mounts, network etc.

Same feedback for other `test.sh` files.